packages/analytics/analytics-chart/src/components/AnalyticsChart.cy.ts

```diff
@@ -723,6 +238,57 @@ describe('<AnalyticsChart />', () => {
     cy.get('[data-testid="no-data-in-report"] .k-empty-state-message').should('contain.text', emptyStateDescription)
   })
 
+  it('doest not render an "Export button" if the datatet is empty', () => {
+    cy.mount(AnalyticsChart, {
+      props: {
+        allowCsvExport: true,
+        chartData: emptyExploreResult,
+        chartOptions: {
+          type: ChartTypes.TIMESERIES_LINE,
+        },
+        chartTitle: 'Requests',
+      },
+    })
+    cy.getTestId('csv-export-button').should('not.exist')
+  })
+
+  it('does not render an "Export" button if chart data is present but prop is set to `false`', () => {
+    cy.mount(AnalyticsChart, {
+      props: {
+        allowCsvExport: true,
```

Comment:
Based on the test description, this should be `false` and the `chartData` prop should be `exploreResult`.

Reply:
Copy / Pasta at its finest. Thanks for pointing out.